spec/app/ics-721-nft-transfer/README.md

```diff
@@ -110,28 +116,24 @@ The sub-protocols described herein should be implemented in a "non-fungible toke
 The NFT asset tracking module should implement the following functions:
 
 ```typescript
-function SaveClass(classId: string, classUri: string) {
+function SaveClass(classId: string, classUri: string, classData: string) {
```

Comment:
1. I think `SaveClass` is better named `SaveOrUpdateClass`
2. `classData` defines the []byte type above, and it is also the []byte type here

Reply:
Resolved.